src/core_plugins/metrics/public/kbn_vis_types/index.js

```diff
@@ -51,14 +51,20 @@ export default function MetricsVisProvider(Private) {
         interval: 'auto',
         axis_position: 'left',
         axis_formatter: 'number',
-        show_legend:1
+        show_legend:1,
+        show_grid: 1
```

Comment:
the diffs are identical on this line, so I think it'll just merge without conflict (at least, it did when I tested this locally after checking out latest master).